scripts/menu.sh

```diff
@@ -123,6 +127,14 @@ function configure_gke() {
   export PROJECT=$(inputbox "Google Cloud" "Existing Project ID" "${PROJECT}")
   export CLUSTER_NAME=$(inputbox "Deepcell" "Cluster Name" "${CLUSTER_NAME:-deepcell}")
   export GKE_BUCKET=$(inputbox "Deepcell" "Bucket Name" "${GKE_BUCKET}")
+  export GKE_COMPUTE_REGION=$(inputbox "Google Cloud" "Compute Region" "us-west1")
+  export GKE_COMPUTE_ZONE=$(inputbox "Google Cloud" "Compute Zone" "us-west1-b")
+  export GKE_MACHINE_TYPE=$(inputbox "Google Cloud" "Node (non-GPU) Type" "n1-standard-2")
+  export GPU_TYPE=$(inputbox "Google Cloud" "GPU type" "nvidia-tesla-p100")
+  export GPU_PER_NODE=$(inputbox "Google Cloud" "GPUs per GPU Node" "1")
+  export GPU_MACHINE_TYPE=$(inputbox "Google Cloud" "GPU Node Type" "n1-standard-4")
+  export GPU_NODE_MIN_SIZE=$(inputbox "Google Cloud" "Minimum Number of GPU Nodes" "0")
+  export GPU_NODE_MAX_SIZE=$(inputbox "Google Cloud" "Maximum Number of GPU Nodes" "4")
```

Comment:
if you pass `4` explicitly, then if the user goes back to re-edit their cluster, it won't use their last known value. 

Suggest instead writing it `"${GPU_NODE_MAX_SIZE:-4}"`

Reply:
I didn't even think of this. Done.